archetype/synapse-archetype-service-rest-get/src/main/resources/archetype-resources/src/main/resources/service-application.properties

```diff
@@ -0,0 +1,16 @@
+ # Copyright 2020 American Express Travel Related Services Company, Inc.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ # in compliance with the License. You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software distributed under the License
+ # is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ # or implied. See the License for the specific language governing permissions and limitations under
+ # the License.
+spring.application.name=Book Rest Service
+spring.main.allow-bean-definition-overriding=true
+server.port=8080
+wavefront.application.name=bookstore
+wavefront.application.service=example-service-rest-book
```

Comment:
This shouldn't be example-service-rest-book.

Reply:
using apiName